src/openmc_heat_driver.cpp

```diff
@@ -100,36 +125,143 @@ void OpenmcHeatDriver::init_mappings()
     }
   }
 
+  // check that all fissionable cells have been mapped to the T/H model.
+  // Note that this does not check for distortions in the model, just that
+  // the surrogate T/H model is sufficiently fine in the pellet region to
+  // capture all fissionable cells in OpenMC.
+  Ensures(openmc_driver_->n_fissionable_cells_ == n_fissionable_mapped);
+
+  // set the number of solid cells before defining mappings for fluid cells
   if (openmc_driver_->active()) {
-    n_materials_ = openmc_driver_->cells_.size();
+    n_solid_cells_ = openmc_driver_->cells_.size();
```

Comment:
`size()` might be too large for an `n_solid_cells_`, which is an int.  Maybe use `gsl::narrow` or `gsl::narrow_cast`?